lib/linguist/languages.yml

```diff
@@ -6147,8 +6147,12 @@ Valve Data Format:
   aliases:
   - keyvalues
   - vdf
+  - valve configuration file
+  - config
+  - cfg
   extensions:
   - ".vdf"
+  - ".cfg"
```

Comment:
Is .cfg actually VDF? From the sample it looks like it is just a  list of commands and not actually VDF.

Reply:
Just made the last couple commits when I saw the suggestion to... without seeing the edits, or thoroughly considering it myself, oops.

I don't believe CFG is the same as VDF, as it doesn't use the same sort of keyvalues structure that's seen in the other VDF style formats. CFG is essentially a batchfile for the Source Engine's console.

Reply:
This is a very generic extension that _will_ clash and cause a lot of misidentification. To prevent this, you will need to identify a lot more users of it and add support for all the most popular users of this extension as part of this PR.

Reply:
This sounds like far more work than it's worth to include a relatively unimportant extension among the other Source Engine languages, so I've reverted this commit https://github.com/github/linguist/pull/5413/commits/a96569049d5640d1d274676bdb7b5f23d864cbe2